mrjones2014/yaclt
# Add a fun ASCII logo to version info

Resolves: #165 

## How to Test

1. Run `yarn build`
2. Run `./dist/index.js --version`
3. It should show the ascii logo
